app/src/main/java/org/mozilla/tv/firefox/channels/ChannelRepo.kt

```diff
@@ -60,7 +60,11 @@ class ChannelRepo(
 
     fun removeChannelContent(tileData: ChannelTile) {
         when (tileData.tileSource) {
-            TileSource.BUNDLED, TileSource.CUSTOM -> {
+            TileSource.CUSTOM -> {
```

Comment:
This is totally fine, but maybe shorter and less duplicate to do:
```
TileSource.BUNDLED, TileSource.CUSTOM -> {
    TelemetryIntegration.INSTANCE.homeTileRemovedEvent(tileData)
    if (tileData.tileSource is TileSource.BUNDLED) addBundleTileToBlackList(tileData.tileSource, tileData.id)
    pinnedTileRepo.removePinnedTile(tileData.url)
}
```

Reply:
Yeah, I was honestly unsure between the two choices.  Wound up going with my gut and making a quick decision, but I'm fine with us reevaluating and changing this one in the future.